### PR TITLE
fix: restore test command and deal with standard deps

### PR DESCRIPTION
**Pull request by amcgee**

### CODE REVIEW

File: cli/config/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (varl) [inline cli/config/jest.config.js]: We may need to set up something for styled-jsx for use with Jest.

File: cli/config/standard-libs.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (varl) [inline cli/config/standard-libs.json]: Any idea if we can get dependabot to look at this file as dependencies?

File: cli/package.json [hunk 3]
```diff
@@ -15,21 +15,31 @@
         "src"
     ],
     "dependencies": {
+        "//": "****** Includes standard libraries from app-shell, so that tests work ******",
```
Comment (varl) [inline cli/package.json]: This is needed for the same reason as we use `dependencies` in `cli-style`, right? To get them installed when a package installs this as a dep?

File: cli/config/jest.config.js (hunk 1 above)
Comment (amcgee) [reply cli/config/jest.config.js]: What do you mean by this?

File: cli/config/standard-libs.json (hunk 2 above)
Comment (amcgee) [reply cli/config/standard-libs.json]: Probably if we made it a package.json, but only the names are actually used at present.  I tried to consolidate the standard deps into a common package so we didn’t commingle them or duplicate them (4 places currently).  I couldn’t get it to work nicely though, so left them as duplicates for now

File: cli/package.json (hunk 3 above)
Comment (amcgee) [reply cli/package.json]: Yeah - they’re not needed for production or development builds which get injected into the she’ll, but when running local tests they’re run outside the shell so they need to be able to find these peerdeps somewhere somewhere

File: cli/config/standard-libs.json (hunk 2 above)
Comment (varl) [reply cli/config/standard-libs.json]: Alright, problems for a bright future to solve.

File: cli/config/jest.config.js (hunk 1 above)
Comment (varl) [reply cli/config/jest.config.js]: https://github.com/zeit/styled-jsx#styled-jsxcss-in-tests

I think @HendrikThePendric and @jenniferarnesen ran into problems using styled-jsx components with Jest and had to do some configuration. I don't remember the details. Could deal with that when we get there.

Comment (amcgee) [reply cli/config/jest.config.js]: Got it.  Tests do seem to work with `styled-jsx` at least with in-line styles - the example installed by `init` has `<style jsx>{...}</style>` and renders properly.  I pass our custom `babel` config explicitly to `babel-jest` which includes the `styled-jsx` transpiler, so maybe that's all that was needed?

Comment (amcgee) [reply cli/config/jest.config.js]: Actually looks like it's an issue with tagged template literals, which aren't working (again) anyway.  Will need to look into that shortly.

**Pull request merged**